addons/account/models/partner.py

```diff
@@ -599,6 +599,21 @@ def _get_company_currency(self):
         compute='_compute_duplicated_bank_account_partners_count',
     )
 
+    property_payment_method_id = fields.Many2one(
+        comodel_name='account.payment.method',
+        company_dependent=True,
+        domain="[('payment_type', '=', 'outbound')]",
+        help="Preferred payment method when paying this vendor. This is used to filter vendor bills"
+             " by preferred payment method to register payments in mass. Use cases: create bank"
+             " files for batch wires, check runs.",
+    )
+
+    property_inbound_payment_method_id = fields.Many2one(
+        comodel_name='account.payment.method',
+        company_dependent=True,
+        domain="[('payment_type', '=', 'inbound')]",
```

Comment:
@chklop Do you have an idea for an helper message here ? Just the same as above but replace paying by being paid or something like that ?